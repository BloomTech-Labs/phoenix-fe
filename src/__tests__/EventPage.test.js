import React from 'react';
import ReactDOM from 'react-dom';
import { BrowserRouter as Router } from 'react-router-dom';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer'
import EventPage from '../components/Event-Page';

it("Renders the event page and matches a snapshot of the rendered page", () => {
  const tree = renderer

    .create(<Router><EventPage /></Router>)
    .toJSON();

    console.log(tree, 'tree')

  expect(tree).toMatchSnapshot()
});
