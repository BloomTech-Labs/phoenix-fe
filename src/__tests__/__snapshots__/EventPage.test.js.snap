// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches a snapshot of event page 1`] = `
Array [
  <h2 />,
  <p />,
  <p>
    Location: 
  </p>,
  <p>
    Starts: 
    
     
    
  </p>,
  <p>
    Ends: 
    
     
    
  </p>,
]
`;

exports[`Renders the event page and matches a snapshot of the rendered page 1`] = `
Array [
  <h2 />,
  <p />,
  <p>
    Location: 
  </p>,
  <p>
    Starts: 
    
     
    
  </p>,
  <p>
    Ends: 
    
     
    
  </p>,
]
`;

exports[`renders 3 'Phoenix' elements 1`] = `
Array [
  <h2 />,
  <p />,
  <p>
    Location: 
  </p>,
  <p>
    Starts: 
    
     
    
  </p>,
  <p>
    Ends: 
    
     
    
  </p>,
]
`;
